PREREQUISITE.md

```diff
@@ -0,0 +1,248 @@
+
+2. Click `File` > `New` > `Import Module...`
+
+<img width="392" alt="module_002" src="https://user-images.githubusercontent.com/27720475/130185349-13eacf92-da1a-47cc-b2fa-cdc3c91b6aaf.png">
+
+<img width="392" alt="module_003" src="https://user-images.githubusercontent.com/27720475/130185355-dfef2041-53c8-4146-905f-39744e499a75.png">
+
+3. Put what you downloaded before in source directory and click `Finish`
+
+<img width="392" alt="module_004" src="https://user-images.githubusercontent.com/27720475/130185365-e4ef05af-60e1-4f37-b0a4-6b87cf2b17b1.png">
+
+<img width="392" alt="module_005" src="https://user-images.githubusercontent.com/27720475/130185374-b42d0860-6b2e-48a7-aa1f-a99eaf52435e.png">
+
+> If Could not get unknown property 'language' for build 'import_Euphony' of type org.gradle.invocation.DefaultGradle. error occur, please add a sentence below in setting.gradle(Project) and click 'Sync Now'
+
+```gradle
+gradle.ext.language = "java"; // or kotlin
+```
+
+<img width="392" alt="module_006" src="https://user-images.githubusercontent.com/27720475/130185379-8f6fb9d8-d4a5-497e-b7ec-514aeb1088a0.png">
+
+> If you want to know more about 'Gradle', [https://www.baeldung.com/gradle-build-settings-properties](https://www.baeldung.com/gradle-build-settings-properties) will be helpful
+
+<img width="392" alt="module_007" src="https://user-images.githubusercontent.com/27720475/130185391-02c82964-299e-484c-80e1-810254dd7070.png">
+
+> If Caused by: groovy.lang.MissingPropertyException: Cannot get property 'signing.keyId' on extra properties extension as it does not exist error occur, please delete the sentence below in euphony/gradle.build and click 'Sync Now'apply from: file('publish.gradle')
+
+<img width="392" alt="module_008" src="https://user-images.githubusercontent.com/27720475/130185404-16a862f0-28ee-4c0e-917c-826fe73d25bb.png">
+
+4. Now, we'll gonna add a dependency. In `File` > `Project Structure`, click `+` button at `Dependencies` > `app`. Then click `Module Dependency`
+
+<img width="392" alt="module_009" src="https://user-images.githubusercontent.com/27720475/130185413-7dc68c68-c756-455a-8829-8d61e128b939.png">
+
+<img width="392" alt="module_010" src="https://user-images.githubusercontent.com/27720475/130185419-68d5df4c-7f71-4d08-b1dd-bea9ddd975c2.png">
+
+After that, you can see a sentence is added in `build.gradle(Module:app)`
+
+```gradle
+dependencies{
+     ...
+      implementation project(path: ':euphony')
+     ...
+}
+
+```
+
+> Path could be different, if you change module name at step 3
+
+---
+
+## 2. Edit your AndroidManifest.xml
+
+Add the following line to the AndroidManifest.xml:
+
+```xml
+<uses-permission android:name="android.permission.RECORD_AUDIO" />
+```
+
+### Request permission to users
+
+Plus, you should consider that the permission model is different depending on the users android version.
```

Comment:
Simple review about missing
I think we missed apostrophe. "users" should be "user's"
```suggestion
Plus, you should consider that the permission model is different depending on the user's android version.
```